gulpfile.js

```diff
@@ -6,37 +6,60 @@ const colors = require(`colors/safe`);
 const del = require(`del`);
 const mustache = require(`gulp-mustache`);
 const rename = require(`gulp-rename`);
+const fs = require(`fs`);
+const path = require(`path`);
 
 const SERVER_ROOT = `build/`;
+const TRANSLATES_PATH = `./src/translate/`;
 
-const translates = [
-  {
-    dest: SERVER_ROOT,
-    url: `./src/translate/en.json`
-  },
-  {
-    dest: `${SERVER_ROOT}pt`,
-    url: `./src/translate/pt.json`
-  },
-  {
-    dest: `${SERVER_ROOT}ru`,
-    url: `./src/translate/ru.json`
-  },
-  {
-    dest: `${SERVER_ROOT}zh-tw`,
-    url: `./src/translate/zh-tw.json`
-  },
-  {
-    dest: `${SERVER_ROOT}tr`,
-    url: `./src/translate/tr.json`
+// GENERATE TRANSLATES
+const translates = [];
```

Comment:
Let's move all code for getting localizations to file? Something like `src/helpers/getLocalizations.js`

Reply:
Great idea - please have a look, will my approach do for you?